Update Quidel doc (#839)
cmu-delphi/delphi-epidata: docs/api/covidcast-signals/quidel.md

Pooling in this way makes estimates available in more geographic areas, as many areas 
report very few tests per day, but have enough data to report when 7 days are considered.

##### Geographical Smoothing

**County, MSA and HRR levels**: In a given County, MSA or HRR, suppose $$N$$ COVID tests 
are taken in a certain time period, $$X$$ is the number of tests taken with positive
results. 

For smoothed signals, after taking the temporal pooling,
- if $$N \geq 50$$, we still use:
$$
p = \frac{100 X}{N}
$$
- if $$25 \leq N < 50$$, we lend $$50 - N$$ fake samples from its parent state to shrink the
estimate to the state's mean, which means:
$$
p = 100 \left( \frac{N}{50} \frac{X}{N} + \frac{50 - N}{50}  \frac{X_s}{N_s} \right) 
$$
where $$N_s, X_s$$ are the number of COVID tests and the number of COVID tests
taken with positive results taken in its parent state in the same time period.
A parent state is defined as the state with the largest proportion of the population 
in this county/MSA/HRR.

Counties with sample sizes smaller than 50 are merged into megacounties for 
the raw signals; counties with sample sizes smaller than 25 are merged into megacounties for

> the smoothed signals.

Reviewer: I would provide either a short description of megacounties or a link to a description. "For example, the megacounty aggregating counties 20001, 20002, 20003, is 20000."

Author: Did check the @krivard, we have a general description of the megacounty [here](https://cmu-delphi.github.io/delphi-epidata/api/covidcast_geography.html#small-sample-sizes-and-megacounties), so no more introduction here.

p = \frac{100 x}{n}
$$

We estimate p across 6 temporal-spatial aggregation schemes:
- daily, at the county level;
- daily, at the MSA (metropolitan statistical area) level;
- daily, at the HRR (hospital referral region) level;
- daily, at the state level;
- daily, at the HHS level;
- daily, at the US national level.

#### Standard Error

We assume the estimates for each time point follow a binomial distribution. The
estimated standard error then is:

$$
\text{se} = 100 \sqrt{ \frac{\frac{p}{100}(1- \frac{p}{100})}{N} } 
$$

#### Smoothing

We add two kinds of smoothing to the smoothed signals:

##### Temporal Smoothing
Smoothed estimates are formed by pooling data over time. That is, daily, for
each location, we first pool all data available in that location over the last 7
days, and we then recompute everything described in the two subsections above. 

> The sample size provided for smoothed signals is different from that for raw signals. 
> The former one is re-definied to be the cumulative sample size in the rolling 7 day 
> window while the latter one is simply the sample size for that corresponding date. 

Reviewer 2: Not needed -- "recompute everything described above" already includes sample size as defined here
```suggestion
```

Pooling in this way makes estimates available in more geographic areas, as many areas 
report very few tests per day, but have enough data to report when 7 days are considered.

##### Geographical Smoothing

**County, MSA and HRR levels**: In a given County, MSA or HRR, suppose $$N$$ COVID tests 
are taken in a certain time period, $$X$$ is the number of tests taken with positive
results. 

For smoothed signals, after taking the temporal pooling,
- if $$N \geq 50$$, we still use:
$$
p = \frac{100 X}{N}
$$

> - if $$25 \leq N < 50$$, we lend $$50 - N$$ fake samples from its parent state to shrink the

Reviewer 2: How do you decide parent state for MSAs/HRRs that cross state boundaries?

Author: The explanation was added. The parent state is defined as the state with the largest proportion of population of that region

estimate to the state's mean, which means:
$$
p = 100 \left( \frac{N}{50} \frac{X}{N} + \frac{50 - N}{50}  \frac{X_s}{N_s} \right) 
$$
where $$N_s, X_s$$ are the number of COVID tests and the number of COVID tests
taken with positive results taken in its parent state in the same time period.

> To be noticed, counties with sample sizes smaller than 50 are merged into megacounties for 
> the raw signals; counties with sample sizes smaller than 25 are merged into megacounties for
> the smoothed signals.

Reviewer 2: ```suggestion
Counties with sample sizes smaller than 50 are merged into megacounties for 
the raw signals; counties with sample sizes smaller than 25 are merged into megacounties for
the smoothed signals.
```

> **State level, HHS level, Nationa level**: the location with fewer than 50 tests are discarded. 
> For the rest of the location with sufficient samples,

Reviewer 2: ```suggestion
**State level, HHS level, National level**: locations with fewer than 50 tests are discarded. For the remaining locations,
```

$$
where $$N_s, X_s$$ are the number of COVID tests and the number of COVID tests
taken with positive results taken in its parent state in the same time period.
A parent state is defined as the state with the largest proportion of the population 

> in this county/MSA/HRR.

Reviewer 2: @ryantibs confirming: if an msa or hrr crosses state boundaries, shrinkage only borrows samples from the state of greatest population within that msa/hrr, and not the others

Reviewer 3: Resolved in teams leads. Sorry for the delay!
